### PR TITLE
Plugins: Add disclaimer

### LINKED ISSUE #5547: Add an obvious disclaimer in the plugins store
Disclaimer: BTCPay plugins are developed by third parties unless the author is `BTCPay` Use at your own risk. [Read More]/#) 

And that can open a pop-up

**Important Notice for BTCPay Plugin Store Users**

**Use at Your Own Risk**: Plugins in this store are developed by independent third parties, not affiliated with the BTCPay core team. These plugins have not undergone review by our team.

**Disclaimer of Responsibility**: The BTCPay team is not liable for any harm, loss, or damage resulting from the installation or use of these plugins. Users assume full responsibility for their choices.

**No Official Endorsement**: Inclusion in the BTCPay Plugin Store does not constitute an endorsement or guarantee of quality, safety, or compatibility by the BTCPay team.

**Due Diligence Advised**: We recommend users exercise caution and conduct their own research or consult the community before installing any plugin.

**Feedback and Reporting**: Should you experience issues with a plugin, please provide feedback or report concerns directly to the respective plugin developers.

I would again leave it up to @dstrukt and @dennisreimann on how this can be achieved UX wise, as I said above, a brief sentence with the ability for users to read more, should be good enough.

### PR DESCRIPTION
**Pull request by dennisreimann**
Closes #5547. Rephrased the original proposed text a bit and included @ndeet's comment.

grafik

### CODE REVIEW

**Comment by pavlenex** [comment]
Good approach. A few notes:

- _Change_:`Important Notice for BTCPay Server Plugin Store Users`  to `Important notice about plugins`
 -  We don't know which licenses plugin developers may use in the future, so let's not assume things upfront.  _Change_ this `Disclaimer of Responsibility: BTCPay Server is not liable for any harm, loss, or damage resulting from the installation or use of these plugins. Users assume full responsibility for their installation, use and maintenance. Usually plugins are distributed under MIT license, which excludes liability and warranty by the plugin developers.` to  `Disclaimer of Responsibility: BTCPay Server contributors or Foundation are not liable for any harm, loss, or damage resulting from the installation or use of the plugins. Users assume full responsibility for their installation, use, familiarity with licensing and terms of service and maintenance.`

File: BTCPayServer/Views/UIServer/ListPlugins.cshtml [hunk 1]
```diff
@@ -113,6 +113,24 @@
 
 <partial name="_StatusMessage" />
 
+<div class="alert alert-warning alert-dismissible mb-4" role="alert">
+    <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close">
+        <vc:icon symbol="close" />
+    </button>
+    <h5 class="alert-heading">Important Notice for BTCPay Server Plugin Store Users</h5> 
+    <p class="mb-0">
+        Plugins are developed by third parties unless the author is BTCPay Server. Use plugins at your own risk.
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIServer/ListPlugins.cshtml:122]:
```suggestion
        Plugins are developed by third parties. Use plugins at your own risk.
```

File: BTCPayServer/Views/UIServer/ListPlugins.cshtml [hunk 2]
```diff
@@ -113,6 +113,24 @@
 
 <partial name="_StatusMessage" />
 
+<div class="alert alert-warning alert-dismissible mb-4" role="alert">
+    <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close">
+        <vc:icon symbol="close" />
+    </button>
+    <h5 class="alert-heading">Important Notice for BTCPay Server Plugin Store Users</h5>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIServer/ListPlugins.cshtml:120]:
```suggestion
    <h5 class="alert-heading">Important notice about plugins</h5> 
```

File: BTCPayServer/Views/UIServer/ListPlugins.cshtml [hunk 3]
```diff
@@ -113,6 +113,24 @@
...
+        <vc:icon symbol="close" />
+    </button>
+    <h5 class="alert-heading">Important Notice for BTCPay Server Plugin Store Users</h5> 
+    <p class="mb-0">
+        Plugins are developed by third parties unless the author is BTCPay Server. Use plugins at your own risk. 
+        <a href="#warning-details" data-bs-toggle="collapse" class="alert-link">Read more</a>
+    </p>
+    <div class="collapse" id="warning-details">
+        <p class="my-3"><strong>Use at Your Own Risk:</strong> Plugins in this store are developed by independent third parties. These plugins have not undergone review by the BTCPay Server team.</p>
+        <p class="mb-3"><strong>Disclaimer of Responsibility:</strong> BTCPay Server is not liable for any harm, loss, or damage resulting from the installation or use of these plugins. Users assume full responsibility for their installation, use and maintenance. Usually plugins are distributed under MIT license, which excludes liability and warranty by the plugin developers.</p>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIServer/ListPlugins.cshtml:127]:
```suggestion
        <p class="mb-3"><strong>Disclaimer of Responsibility:</strong> BTCPay Server contributors or Foundation are not liable for any harm, loss, or damage resulting from the installation or use of the plugins. Users assume full responsibility for their installation, use, familiarity with licensing and terms of service and maintenance.</p>
```

File: BTCPayServer/Views/UIServer/ListPlugins.cshtml [hunk 4]
```diff
@@ -113,6 +113,24 @@
...
+        <p class="mb-3"><strong>No Official Endorsement:</strong> Inclusion in the list of BTCPay Server plugins does not constitute an endorsement or guarantee of quality, safety, or compatibility by BTCPay Server.</p>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIServer/ListPlugins.cshtml:128]:
```suggestion
        <p class="mb-3"><strong>No Official Endorsement:</strong> Inclusion in the list of BTCPay Server plugins does not constitute an endorsement or guarantee of quality, safety, or compatibility.</p>
```

**Comment by pavlenex** [comment]
Looks a bit odd with yellow text on white background? 
Screenshot 2023-12-12 at 15 24 11
Screenshot 2023-12-12 at 15 24 40

**Comment by pavlenex** [comment]
@Kukks can you jump in on this one please?
So if enabled, third-parties will be able to receive payments to your store ,but not spend it? Should we provide any warring here?
Screenshot 2023-12-12 at 15 27 51

**Review by Kukks — APPROVED** [review]

**Comment by dennisreimann** [comment]
On more thing we need to make people aware of — ideally in the first sentence — is the fact that they need to update/maintain plugins independent of/in addition to BTCPay Server. 

"Plugins are developed and distributed by third parties. They need to be updated and maintained in addition to BTCPay Server. Use plugins at your own risk."

**Comment by dstrukt** [comment]
> "Plugins are developed and distributed by third parties. They need to be updated and maintained _often_ _regularly_ in addition to BTCPay Server. Use plugins at your own risk."

Was just a thought, should we include a frequency as well, to be clear it's a semi-regular thing?

**Comment by dennisreimann** [comment]
Updated and replaced the screenshot with the now current version.

**Pull request merged**